container-registry/aws/ecr/versions.tf

```diff
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 5.4.0"
+      version = ">= 5.61"
```

Comment:
Do you really need to upgrade the provider constraint?

Reply:
yes, newer versions of the eks module depends on it